### PR TITLE
Blue Whale (GSI-1266)

### PR DESCRIPTION
**Pull request by Cito**

### CODE REVIEW

**Review by SilverLinings89 — DISMISSED** [review]

**Comment by SilverLinings89** [comment]
Looks good to me.

File: 64-blue-whale/technical_specification.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+**Epic Type:** Implementation Epic
+
+Epic planning and implementation follow the
+[Epic Planning and Marathon SOP](https://docs.ghga-dev.de/main/sops/sop001_epic_planning.html).
+
+## Scope
+
+### Outline:
+
+The goal of this epic is to implement all the features from the current React based GHGA Data Portal in Angular, using the baseline application that has been built in the [Hippopotamus](../60-hippopotamus/technical_specification.md) as the starting point.
```
Comment (ac-jorellanaf) [inline 64-blue-whale/technical_specification.md:12]:
```suggestion
The goal of this epic is to implement all the features from the current React-based GHGA Data Portal in Angular, using the baseline application that has been built in the [Hippopotamus](../60-hippopotamus/technical_specification.md) as the starting point.
```
And below

File: 64-blue-whale/technical_specification.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+We have also added rudimentary unit tests and end-to-end tests. While implementing the remaining functionality in the following four steps, basic tests should be included together with the implemented functionality. As a last step, the implementation and tests should be refined.
+
+### Step 1: Catalog functionality
+
+In this epic, we first implement functions from the GHGA Catalog.
+
+- [ ] Metadata search service (complete)
+- [ ] Dataset browser (complete)
+	- [ ] Result list
```
Comment (ac-jorellanaf) [inline 64-blue-whale/technical_specification.md:59]: What is there left for the results list itself? The summary is the thing missing no?

File: 64-blue-whale/technical_specification.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+- [ ] IVA manager
+	- [ ] List
+	- [ ] Filter
+	- [ ] (Re)create code
+	- [ ] Invalidation
+	- [ ] Confirm transmission
+
+### Polish and improve test coverage
+
+In the last step, we polish everything (edge cases, layout etc.) and increase the test coverage from unit tests and end-to-end tests. The latter should cover the complete user journey from dataset discovery to dataset download, but they should use the static (stateless) mock backend that is part of the frontend repository.
```
Comment (ac-jorellanaf) [inline 64-blue-whale/technical_specification.md:114]: We may perhaps want to include small-screen responsiveness too in this section, which might take half a sprint for all the pages and trying to deal with tailwind and it not overriding material styles.

**Review by ac-jorellanaf — COMMENTED** [review]
I only have a few comments, but besides that pretty good.

File: 64-blue-whale/technical_specification.md (hunk 2 above)
Comment (Cito) [reply 64-blue-whale/technical_specification.md:59]: That's true, you did that already including pagination. Maybe we should add some tests and a loading indicator or ghost elements while waiting for results, because this can sometimes be slow.

File: 64-blue-whale/technical_specification.md (hunk 1 above)
Comment (Cito) [reply 64-blue-whale/technical_specification.md:12]: Changed this.

File: 64-blue-whale/technical_specification.md (hunk 2 above)
Comment (Cito) [reply 64-blue-whale/technical_specification.md:59]: Adapted this.

**Review by SilverLinings89 — APPROVED** [review]
